src/main/java/io/microconfig/osdf/install/BashrcInstaller.java

```diff
@@ -37,15 +37,20 @@ public void prepare() {
     }
 
     private String content() {
-        String newEntry = "PATH=$PATH:" + paths.bin() + "/";
+        String newEntry = "PATH=$PATH:" + paths.bin() + "/" + "\n";
+        String additionalEntry = "\n" + "source .osdf_completion" + "\n";
```

Comment:
why not use single variable newEntry?

Reply:
necessary, because entry order is important